### PR TITLE
Add new developer/libraries  `@khofaai`

### PR DESCRIPTION
**Pull request by khofaai**
Add @khofaai a Moroccan developer,

### CODE REVIEW

**Comment by khofaai** [comment]
@akiyamaSM

File: readme.md [hunk 1]
```diff
@@ -50,7 +50,8 @@
   - [Amine Hakkou](https://www.hakkou.me/) - Javascript, React Native ....
   - [Abderrahim SOUBAI-ELIDRISI](https://www.soubai.me/) - Javascript, Node.js , Angular ....
   - [Meriem Zaid](https://imeriem.com/) - Javascript, Python, React ....
-
+  - [Anas El Alami](https://beta.anaselalami.com/) - Javascript, NodeJS, Vuejs, ...
```
Comment (yjose) [inline readme.md]: Can you please add your blog URL instead of your portfolio?

Comment (khofaai) [reply readme.md]: Unfortunately i don't have a blog for the moment, I'll remove my portfolio link  👍

Comment (yjose) [reply readme.md]: sorry for that, I think it's time to start 😄

Comment (khofaai) [reply readme.md]: yeah, i have only two medium posts for now, at least i can start adding them as a fresh `restart` 👍 , thanks

File: readme.md [hunk 2]
```diff
@@ -75,6 +75,8 @@
 - [daktilo](https://github.com/kronik3r/daktilo) - A Jekyll theme with a minimal design inspired by typewriters. by [@kronik3r](https://github.com/kronik3r)
 - [v-odometer](https://github.com/JefferyHus/v-odometer) - Vue.js component to smoothly transition numbers with ease. by [@jefferyhus](https://github.com/JefferyHus)
 - [es6-crawler-detect](https://github.com/JefferyHus/es6-crawler-detect) - ES6 module to help you detected and block bots, crawlers and spiders. by [@jefferyhus](https://github.com/JefferyHus)
+- [vue-dropify](https://www.npmjs.com/package/vue-dropify) - Dropify build for vuejs framework. by [@khofaai](https://github.com/khofaai)
```
Comment (yjose) [inline readme.md]: can you use GitHub repo URLs instead of npm package url .

Comment (khofaai) [reply readme.md]: Done,

**Pull request merged**